packages/twenty-front/src/modules/activities/states/tableSortFamilyState.ts

```diff
@@ -0,0 +1,29 @@
+import { SerializableParam } from 'recoil';
+
+import { OrderBy } from '@/object-metadata/types/OrderBy';
+import { createFamilyState } from '@/ui/utilities/state/utils/createFamilyState';
+
+interface TableSortState {
+  fieldName: string;
+  orderBy: OrderBy | null;
+}
+
+type TableSortFamilyKey = {
```

Comment:
tableId: string; fieldName: string. Why initial ? 

And 

type TableSortValue = OrderBy | null;